docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+Upload Rights
+#############
+
+Those who are allowed to upload the kernel to the Ubuntu archive take their
+responsibility very seriously.
+
+To obtain per package upload rights for the Ubuntu kernel, you can apply to
+become a member of the ubuntu-kernel-uploaders_ team in Launchpad.
+
+People can join this team only after going through a thorough application and
+review process. The sections below describe the general member profile of the
+ubuntu-kernel-uploaders_ team, as well as the application process.
+
+Member Profile
+==============
+
+Below is the general profile for those having per package upload rights for the
+Ubuntu kernel.
+
+- Members of the ubuntu-kernel-uploaders_ Launchpad team
+
+- Generally have commit access to the `Ubuntu kernel git repositories`_
+
+- Actively follow and participate in discussions and patch reviews on the
+  `Ubuntu kernel-team mailing list`_
+
+- Are collectively responsible for the maintenance of packages in the Ubuntu
+  kernel package set for all supported releases as well as the development
+  release
+
+- Have a strong working knowledge of kernel packaging concepts and techniques,
+  refined through experience
+
+- Have a strong working knowledge of Ubuntu project procedures, especially
+  those related to the release process and support commitments, and an
+  understanding of the reasons why they exist
+
+- Have a history of substantial direct contributions to the distribution and
+  particularly the kernel related packages
+
+- Feel a sense of personal responsibility for the quality of Ubuntu releases
+  and for the satisfaction of Ubuntu users
+
+- Exercise great care in their work, with the understanding that their efforts
+  have a direct impact on others, including:
+
+  - every Ubuntu user
+  - the Ubuntu release team
+  - corporate partners who provide support for Ubuntu
+
+Application Process
+===================
+
+As alluded to in the member profile above, membership consideration for the
+ubuntu-kernel-uploaders team adheres to a strict policy. Anyone considering
+applying should meet **all** of the criteria listed below:
+
+1. Have a history of substantial and direct contribution to the distribution;
```

Comment:
https://github.com/canonical/kernel-docs/pull/36/files#r1872672861

I think both statements should be reconciled

Reply:
Hi @gagath ,

Maybe clean up the Member profile section a little so the wording matches how it's listed in the Application process section as a criteria (where necessary).

And in the Application process section, update it to something like:

```
Anyone considering applying should align with the general profile outlined in the previous section and meet the criteria listed below:
```

And only list the 'additional' criteria to supplement the general profile expectations.

Thoughts?